### PR TITLE
Intly 4532

### PR DESCRIPTION
**Pull request by briandooley**

### CODE REVIEW

File: walkthroughs/1-ionic-showcase/walkthrough.adoc [hunk 1]
```diff
@@ -132,8 +130,8 @@ In this task, we will protect both the frontend and the backend of the {data-syn
 
 === Configure and protect the frontend app
 
-. Make sure you have access to a realm in the {customer-sso-name}. If you don't, you can ask your customer-admin user to create one for you. 
-. Login to the realm in the {customer-sso-name}.
+. Make sure you have access to a realm in the link:{sso-realm-url}[{customer-sso-name}, window="_blank"]. If you do not, you can ask your customer-admin user to create one for you.
```
Comment (pwright) [inline walkthroughs/1-ionic-showcase/walkthrough.adoc]: this seems wrong?

Comment (briandooley) [reply walkthroughs/1-ionic-showcase/walkthrough.adoc]: Still a draft PR, not ready for review yet.

**Comment by briandooley** [comment]
@pwright Can you review please?

@jhellar I've noticed, when I attempted to perform step 4 of this walkthough, that the aerogear app freezes when I run it on safari or chrome on IOS 13 in airplane mode.  Can QE take a look ?

**Comment by jhellar** [comment]
@briandooley ok, will take a look

**Comment by jhellar** [comment]
@briandooley I've reported the issue in https://issues.redhat.com/browse/INTLY-4813

File: walkthroughs/1-ionic-showcase/walkthrough.adoc [hunk 2]
```diff
@@ -108,14 +106,14 @@ Verify that you followed each step in the procedure above.  If you are still hav
 The showcase app demonstrates the key capabilities provided by the {data-sync-name} framework.
 It can be run either as a hybrid mobile application, or a Progressive Web App (PWA).
 
-. Go to link:{route-ionic-showcase-server-host}[the client app, window="_blank"]
+. Go to the link:{route-ionic-showcase-server-host}[newly created client app, window="_blank"]
```
Comment (pwright) [inline walkthroughs/1-ionic-showcase/walkthrough.adoc]:
```suggestion
. Navigate to the link:{route-ionic-showcase-server-host}[Data Sync application user interface, window="_blank"]
```

File: walkthroughs/1-ionic-showcase/walkthrough.adoc (hunk 1 above)
Comment (pwright) [reply walkthroughs/1-ionic-showcase/walkthrough.adoc]: don't think we need this step? it's like 'make sure RH haven't messed up'

File: walkthroughs/1-ionic-showcase/walkthrough.adoc [hunk 3]
```diff
@@ -48,9 +49,8 @@ image::images/arch.png[architecture, role="integr8ly-img-responsive"]
 * `mosquitto-mqtt-broker` - a message broker
 * `postgresql` - a database
 
-Then you can use the build-in GraphQL playground to interact with the server API:
-. Navigate to the link:{route-ionic-showcase-server-host}/graphql[GraphQL playground, window="_blank"] and remove any text in the left pane of the screen. Paste the following query/mutation into the left pane of the screen:
-+
+You can use the build-in GraphQL playground to interact with the server API. Navigate to the link:{route-ionic-showcase-server-host}/graphql[GraphQL playground, window="_blank"] and remove any text in the left pane of the screen. Paste the following query/mutation into the left pane of the screen:
```
Comment (pwright) [inline walkthroughs/1-ionic-showcase/walkthrough.adoc]:
```suggestion
. Navigate to the link:{route-ionic-showcase-server-host}/graphql[GraphQL playground, window="_blank"] and replace the text in the left pane of the screen with the following query and mutation::
```

File: walkthroughs/1-ionic-showcase/walkthrough.adoc [hunk 4]
```diff
@@ -48,9 +49,8 @@ image::images/arch.png[architecture, role="integr8ly-img-responsive"]
 * `mosquitto-mqtt-broker` - a message broker
 * `postgresql` - a database
```
Comment (pwright) [inline walkthroughs/1-ionic-showcase/walkthrough.adoc:50]:
```suggestion
* `postgresql` - a database
+
Once the pods are running, you can use the GraphQL playground to interact with the server API as described in the next step.
```

File: walkthroughs/1-ionic-showcase/walkthrough.adoc [hunk 5]
```diff
@@ -132,63 +130,78 @@ In this task, we will protect both the frontend and the backend of the {data-syn
...
 . Create a new client for the backend:
 .. Choose *Clients* under *Configure* in the menu.
 .. Click on the *Create* button on the top-right corner to start creating a new client.
-.. Enter `showcase-app-server` for Client ID.  Do not change the rest of the fields.
+.. Enter `{user-username}-server` for Client ID.  Do not change the rest of the fields.
 .. Save the client. You should see the *Settings* page of the client.
-.. Change the `Access Type` to `bearer-only` and save again.
-.. Click on the *Installation* tab, and select `OIDC JSON` format. Copy the content or use the `Download` button to save the configuration file to a directory on the computer.
+.. Change the `Access Type` to `bearer-only`
+.. Click on the *Installation* tab, and select `Keycloak OIDC JSON` format. Use the `Download` button to save the configuration file to a directory on the computer.
```
Comment (pwright) [inline walkthroughs/1-ionic-showcase/walkthrough.adoc]:
```suggestion
.. Click on the *Installation* tab, and select `Keycloak OIDC JSON` format. Use the `Download` button to save the `keycloak.json` configuration file.
```

File: walkthroughs/1-ionic-showcase/walkthrough.adoc [hunk 6]
```diff
@@ -197,7 +210,7 @@ showcase-server-idm-config
 ----
 keycloak.json
 ----
-.. For *Value*, click *Browse* and load the `keycloak.json` file that you downloaded previously.
+.. For *Value*, click *Browse* and load the json file that you downloaded previously.
```
Comment (pwright) [inline walkthroughs/1-ionic-showcase/walkthrough.adoc]: switched from first order list to second order for no good reason?

File: walkthroughs/1-ionic-showcase/walkthrough.adoc [hunk 7]
```diff
@@ -247,6 +262,11 @@ Verify that you followed each step in the procedure above.  If you are still hav
 To explore data sync features, you should run multiple instances of the showcase app using different browsers.
 For example, use the browser on your mobile device as well as using the browser on your laptop.
 
+To get the url of your app:
+
+. Navigate to the {openshift-host}/console/project/{walkthrough-namespace}/browse/routes[Routes page of the OpenShift console]
```
Comment (pwright) [inline walkthroughs/1-ionic-showcase/walkthrough.adoc]: can replace all this with 

Navigate to the link:{route-ionic-showcase-server-host}[Data Sync user interface]

**Review by pwright — CHANGES_REQUESTED** [review]
some comments

**Comment by briandooley** [comment]
@pwright Changes made, review again pur-lease?

**Review by pwright — APPROVED** [review]

**Pull request merged**